x-pack/plugins/observability_solution/synthetics/public/apps/embeddables/status_overview/status_overview_embeddable_factory.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import React, { useEffect } from 'react';
+import { DefaultEmbeddableApi, ReactEmbeddableFactory } from '@kbn/embeddable-plugin/public';
+import {
+  initializeTitles,
+  useBatchedPublishingSubjects,
+  fetch$,
+  PublishesWritablePanelTitle,
+  PublishesPanelTitle,
+  HasEditCapabilities,
+  SerializedTitles,
+} from '@kbn/presentation-publishing';
+import { BehaviorSubject, Subject } from 'rxjs';
+import type { StartServicesAccessor } from '@kbn/core-lifecycle-browser';
+import { SYNTHETICS_OVERVIEW_EMBEDDABLE } from '../constants';
+import { ClientPluginsStart } from '../../../plugin';
+import { StatusOverviewComponent } from './status_overview_component';
+
+export const getOverviewPanelTitle = () =>
+  i18n.translate('xpack.synthetics.statusOverview.displayName', {
+    defaultMessage: 'Synthetics Status Overview',
+  });
+
+export interface OverviewEmbeddableState extends SerializedTitles {
+  tags: string[];
+}
+
+export type StatusOverviewApi = DefaultEmbeddableApi<OverviewEmbeddableState> &
+  PublishesWritablePanelTitle &
+  PublishesPanelTitle &
+  HasEditCapabilities;
+
+export const getStatusOverviewEmbeddableFactory = (
+  getStartServices: StartServicesAccessor<ClientPluginsStart>
+) => {
+  const factory: ReactEmbeddableFactory<
+    OverviewEmbeddableState,
+    OverviewEmbeddableState,
+    StatusOverviewApi
+  > = {
+    type: SYNTHETICS_OVERVIEW_EMBEDDABLE,
+    deserializeState: (state) => {
+      return state.rawState as OverviewEmbeddableState;
+    },
+    buildEmbeddable: async (state, buildApi, uuid, parentApi) => {
+      const { titlesApi, titleComparators, serializeTitles } = initializeTitles(state);
+      const defaultTitle$ = new BehaviorSubject<string | undefined>(getOverviewPanelTitle());
+      const tags$ = new BehaviorSubject(state.tags);
```

Comment:
Is this used anywhere? Later on when you call `useBatchedPublishingSubjects`, I would expect to retrieve the tags value there and pass it to the component. Shall we then remove it, since it is not used?